Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -17,6 +17,10 @@ require({
         name : 'Source',
         location : '.'
     }, {
+        name: 'CesiumUnminified',
+        location : '../../Build/CesiumUnminified',
```

Comment:
Careful here.  On line `6` above, `baseUrl` is set to be the `Source` folder, and your build script already moves this one level higher in the transformed version of this file.  So, line `21` here has one too many parent-folder references, I believe.  (On my machine I serve Cesium from a URL path, so I get broken when there are too many `..` instances).

Reply:
I think this change will fix it for me, at least this is the only one I see that looks like it goes up too many levels.  I didn't try making the change myself.

Reply:
The location reflects what it should be when the app is copied to the `Build/Apps` folder. There is no reason to use this location in the unbuilt version, so I used the built path so it didn't need to be modified in the build process, but that not very clear.

Reply:
But take a look at what actually gets built.  The built version of this file, on line 6, you can see `baseUrl` has been transformed by the build from `../../Source` to `../../../Source`.  So, this URL is then `${built Sandcastle folder}/../../../Source/../../Build/CesiumUnminified`.  In this case, it correctly finds the `Source` folder, but then it goes "too high" above `Source` when looking for `Build`.

This works on your machine because you bump into the root path there and it doesn't matter how many extra `..`s there are.  But on my machine, I install Cesium in a sub-folder `/Cesium`, partly because it is the only way to catch mistakes like these.

Reply:
OK, I see!